scripts/udp_bidirectional_flow.py

```diff
@@ -0,0 +1,138 @@
+import snappi
+from snappi_test.tests import utils
+
+
+def udp_bidirectional_flow():
+    """
+    This script does following:
+    - Send 1000 packets back and forth between the two ports at a rate of
+      1000 packets per second.
+    - Validate that total packets sent and received on both interfaces is as
+      expected using port metrics.
+    - Validate that captured UDP packets on both the ports are as expected.
+    """
+    # create a new API instance where location points to controller
+    # location = "https://<tgenip>:<443(or)11009(or)port_set_by_user>"
+    # ext = ixnetwork (or) none for Athena
```

Comment:
do not mention athena anywhere. use `ixia-c` instead.
moreover, simplify the comments by just specifying what should be the location and ext for `ixia-c`, `ixnetwork` and `trex`

feel free to leave out the input for trex as blank

Reply:
# create a new API instance where location points to controller
    # ixia-c, trex:
    #   location = "https://<tgen-ip>:<port>"
    # ixnetwork
    #   location = "https://<tgen-ip>:<port>", ext="ixnetwork"

Reply:
is it fine with the above way?